src/Multiplier.php

```diff
@@ -203,7 +203,7 @@ protected function isValidMaxCopies(): bool
 	public function validate(?array $controls = null): void
 	{
 		/** @var Control[] $controls */
-		$controls = $controls ?? iterator_to_array($this->getComponents(false, Control::class));
+		$controls = $controls ?? iterator_to_array($this->getComponents(true));
```

Comment:
Since, the `noValidate` is used only here: https://github.com/contributte/forms-multiplier/pull/69/files#diff-bbd929c6f77fe505e5255934192003ffef1ca43d2ecd1b908083449269892acaR285

I think, that it should be here without parameter `deep` (as it was before refactoring, which introduces this bug). So It has default value of `false`. There is no need to have deep here I think.

Reply:
I dont think I've tried version w/o parameter but I'll take your word for it :) pushed the change just now